### ISSUE TITLE
MEI import: handling of StaffDef incomplete

### ISSUE THREAD

**Issue by heinzr** [cuthbertLab/music21#771]
StaffDef elements are imported correcctly.

As is described in StaffDefFromElements, keySig and meterSig are currently not supported.

However, the Verovio MusicXML converter uses keySig and MeterSig for the MEI files it creates, which means that 'key' and 'meter' will currently not imported into music21.

**Comment by mscuthbert** [comment]
yes -- the MEI importer written by the McGill-ELVIS team has most elements of MEI missing.  It's not something the core team (me) is working on now, so we'll need to pair it with a PR to make progress.
